apps/topcryptoprices/top_crypto_prices.star

```diff
@@ -0,0 +1,331 @@
+"""
+Applet: Top Crypto Prices
+Summary: Top Crypto USD Prices
+Description: This app shows the latest prices (USD) and price changes for the top cryptocurrencies. It is a very simple app, but does exactly what I need and what I bought the Tidbyt for in the first place. Prices are updated through the free Coingecko API at a maximum rate of once per minute.
+Author: playak
+
+Version: 1.1
+Date: 2022-11-09
+Comments: Written to display the price info of top cryptocurrencies on a Tidbyt screen. We manually exclude stable coins and wrapped coins (so we only need one API call).
+Copyright: © 2022 Jeroen Houttuin, Playak - jeroen@playak.com - https://playak.com
+"""
+
+# CONFIG
+TTL = 60  # time to live for the API cache. Be nice to the free Coingecko API provider!
+PEGGEDCOINS = " USDT USDC BUSD STETH DAI FRAX WBTC USDP GUSD TUSD CUSDC USDD CUSDT USDD PAXG XAUT "  # ignore pegged tokens
+DEFAULTDELAY = "2000"
+DEFAULTNUMCOINS = "5"
+DEFAULTEXCLUDES = "SHIB BSV"  # SHIB because it messes up formatting and I cannot solve it. BSV for obvious reasons.
+DEFAULTCURRENCY = "USD"
+DEFAULTEXCLUDEPEGGED = True
+
+# LOAD MODULES
+load("render.star", "render")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("cache.star", "cache")
+load("math.star", "math")
+load("schema.star", "schema")
+
+# INIT
+credits = ["Data:Coingecko", "Code:Playak"]  # Credits required for using free Coingecko API
```

Comment:
Giving credit in the app description should be sufficient to meet the API terms, and could perhaps make for a better user experience?

Reply:
Good idea. I made that configurable now, as an extra option. Default is to not show the credits on the Tidbyt.